lib/profiles/MEM-SUBM.js

```diff
@@ -1,7 +1,7 @@
 
 exports.name = "MEM-SUBM";
 exports.config = {
-    status:             "SUBM"
+    status:             "MEM-SUBM"
```

Comment:
The short status for member submission is `SUBM` (see recent submissions at https://www.w3.org/Submission/)